packages/ui5-library-generator/package.json

```diff
@@ -0,0 +1,59 @@
+{
+    "name": "@sap-ux/ui5-library-sub-generator",
```

Comment:
The folder should be aligned with the name of the module e.g. `ui5-library-sub-generator`

Reply:
Thanks I've updated